wp-page-fm-player.php

```
@@ -72,20 +72,16 @@
         continue;
     }
 
-    // Normalize legacy ether values that include the unit.
-    $value = trim((string)($row['radio_frequenties_frequentie'] ?? ''));
-    if ($medium === 'Ether') {
-        $value = trim(preg_replace('/\s*FM$/i', '', $value));
-    }
-
+    // Editors sometimes type "Kanaal 914" or "105.8 FM" despite the field asking for the number
+    // only; the group heading already names the medium, so drop that decoration.
+    $value = trim(preg_replace('/^\s*kanaal\s+|\s*fm\s*$/i', '', (string)($row['radio_frequenties_frequentie'] ?? '')));
```

PHPStan (2.2.8)

[warning] Parameter `#1` $string of function trim expects string, string|null given.


(argument.type)